herald.yaml

```diff
@@ -13,6 +14,14 @@ task_store_backend:
   credentials:
     accessKeyId: minio
     secretAccessKey: password
+
+service_accounts:
+  - name: "system:serviceaccount:dev-s3-herald:default"
+    buckets:
```

Comment:
I like it! As a future improvement, the bucket list could use a regex with a {namespace} and {serviceaccount} tag so the structure could be "dynamic", e.g. `{serviceaccount}-.*` let the service account do anything with the `dev-s3-herald-` prefix